osmaxx-py/osmaxx/excerptexport/static/excerptexport/scripts/formValidation.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+(function() {
+    var FormValidator = function() {
+        this.submitButton = document.getElementById('submit-id-submit');
+        this.extractionFormatCheckboxes = document.querySelectorAll('#div_id_formats input[type="checkbox"]');
+
+
+        this.validity = {
+            'extractionFormats': false
+        };
+
+        this.isFormValid = function() {
+            var isValid = true;
+            Object.keys(this.validity).forEach(function(key){
+                isValid = isValid && this.validity[key];
+            }.bind(this));
+            return isValid;
+        };
```

Comment:
Is `arr.every` working for objects? 
My opinion: Polyfill would be a bit overdesigned for this case.